components/SettingsComponent.vue

```diff
@@ -51,6 +51,10 @@ const d = Object.fromEntries(Object.entries(data).filter(([k, v]) =>
   color: var(--vp-custom-block-danger-text);
   background-color: var(--vp-custom-block-danger-bg);
 }
+
+th {
```

Comment:
can this be
```
.dovecotSettings th
```
please?

Reply:
It can, yes.
But the style is scoped, so it wouldn't leak to other components anyway.